docs/_posts/2024-02-27-whats-new-in-cdi41.md

```diff
@@ -0,0 +1,115 @@
+- `InvokerFactory.createInvoker()`, if using a Build Compatible Extension
+
+to obtain an `InvokerBuilder`. Calling `build()` on it will produce
+
+- an `Invoker` directly, if using a Portable Extension
+- an `InvokerInfo` that materializes as an `Invoker` at application runtime, if using a Build Compatible Extension
+
+The `Invoker` then needs to be stored somewhere until the framework actually needs it.
+When the time comes to invoke the method, the framework takes the `Invoker`, obtains an instance of `MyService` and all the parameter values, and calls
+
+```java
+invoker.invoke(myService, new Object[] { "world" })
+```
+
+The return value is `"Hello world!"`, as expected.
+
+### Lookups
+
+It is fairly common that some of the parameters the method accepts are CDI beans.
+It is possible to configure the `InvokerBuilder` to resolve and instantiate a correct value for such parameter automatically.
+In fact, this is possible also for the instance on which the method should be invoked.
+The `InvokerBuilder` has methods `withInstanceLookup()` and `withArgumentLookup()` to configure that.
+
+Returning back to the example above, if the CDI extension called `withInstanceLookup()` before calling `InvokerBuilder.build()`, it wouldn't have to obtain the instance of `MyService` manually.
+The invoker would do it automatically.
+The invocation would just use `null` as the instance value:
+
+```java
+invoker.invoke(null, new Object[] { "world" })
+```
+
+The result is still the same: `"Hello world!"`.
+
+## Other Improvements
+
+There's a couple of quality of life improvements in this release, too.
+
+Interceptors can now access the interceptor binding annotations from `InvocationContext`, using methods `getInterceptorBinding()` and `getInteceptorBindings()`.
+This is technically a change in the Interceptors specification, but is most useful for CDI programmers, so we mention it here as well. 
+
+Producer methods and fields can now declare a `@Priority` directly (or obtain a priority from a stereotype); they no longer have to rely on the priority value of their declaring bean.
+When both the producer and its declaring bean have a priority, the producer's priority wins.
+
+The newly added methods `BeanContainer.isMatchingBean()` and `isMatchingEvent()` provide programmatic access to the typesafe resolution and observer resolution rules.
+This is useful for frameworks that want to perform the same matching as CDI, but don't want to reimplement the rules themselves.
+
+The newly added method `BeanContainer.getContexts()` returns context objects of all registered contexts for given scope, regardless of whether they are active or inactive.
+This is useful for authors of Build Compatible Extensions, who don't provide a context object to the container directly.
+Instead, they provide a _class_ of the context object, and they need to use this method to gain access to the context object which the CDI container created internally.
```

Comment:
created here should be creates or 'will be created'

Reply:
Good point, changed to _creates_.